### ISSUE TITLE
Test focusable elements in navigation

### ISSUE THREAD

**Issue by kimisgold** [omeka/theme-thanksroy#55]
Please test that the following elements can be tabbed to via the keyboard:

* At high zoom levels (200%+), the "button" that toggles the condensed navigation. It previously appeared but could not receive focus, due to being a div element. It is now appropriately a button element.
* The advanced search options toggle after the text input. It was focusable but lacked the visual cues, like the dark glow/shadow.

These need to be tested across desktop browsers (Safari for macOS, Google Chrome, Mozilla Firefox, and Microsoft Edge)

**Comment by katknow** [comment]
This all worked perfectly for me on Safari, Chrome, Firefox, and Edge.
